content/applications/essentials/in_app_purchase.rst

```diff
@@ -1,57 +1,186 @@
+   to the contact, and logs what was sent in the *chatter* of the contact's form.
+
+   Upon sending the SMS message, the prepaid credits for the *SMS* |IAP| service are automatically
+   deducted from the existing credits. If there are not enough credits to send the message, Odoo
+   prompts the user to purchase more.
+
+.. seealso::
+   For more information on how to use various |IAP| services, and for more in-depth instructions
+   related to SMS functionality in Odoo, review the documentation below:
+
+   - :doc:`Lead mining </applications/sales/crm/acquire_leads/lead_mining>`
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an |IAP| service is used, the prepaid credits for that service are spent. Odoo prompts
+the purchase of more credits when there are not enough credits left to continue using a service.
+Email alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in *Packs* from the `Odoo IAP Catalog
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
+   denominations of:
+
+   - :guilabel:`Starter Pack`: 10 credits
+   - :guilabel:`Standard Pack`: 100 credits
+   - :guilabel:`Advanced Pack`: 500 credits
+   - :guilabel:`Expert Pack`: 1,000 credits
+
+   .. image:: in_app_purchase/packs.png
+      :align: center
+      :alt: Four different packs of credits for the SMS IAP service.
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+
+   For more information, refer to the :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
+
+.. _iap/buying_credits:
+
+Buy credits
+-----------
+
+If there are not enough credits to perform a task, the database automatically prompts the purchase
+of more credits.
+
+Users can check the current balance of credits for each service, and manually purchase more credits,
+by navigating to the :menuselection:`Settings app --> Contact --> Odoo IAP --> View My Services`.
+
+Doing so reveals an :guilabel:`IAP Account` page, listing the various |IAP| services in the
+database. From here, click an |IAP| service to open its :guilabel:`Account Information` page, where
+additional credits can be purchased.
+
+Manually buy credits
+~~~~~~~~~~~~~~~~~~~~
+
+To manually buy credits in Odoo, follow these steps:
+
+First, go to the :menuselection:`Settings application` and type `IAP` in the search bar. Under the
+Contacts section, where it says :guilabel:`Odoo IAP`, click :guilabel:`View My Services`.
```

Comment:
I like that these are different steps- I just worry about cinfusing the rader. Maybe state that they cna either search or scroll down to the section?